x-pack/plugins/security/server/authentication/authenticator.ts

```diff
@@ -80,13 +80,6 @@ export interface ProviderLoginAttempt {
    * Login attempt can have any form and defined by the specific provider.
    */
   value: unknown;
-
```

Comment:
__note:__ reducing API surface. This workaround was introduced exactly for this use case and we don't need it anymore.